.packer/template.pkr.hcl

```diff
@@ -0,0 +1,38 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 0.0.2"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+source "amazon-ebs" "source" {
+  ami_name      = "Deep Learning AMI Neuron PyTorch - {{timestamp}}"
+  ami_users     = [
+    "844815912454",
```

Comment:
This allows any account to have access to these AMIs.

I haven't quite worked this out yet but it feels like this might be necessary in the future

Probably we want to build the AMI in dev and generate/train our models with it when there's a version change in the search-config.toml and then persist it to all of the places we build images so potentially this just isn't needed.

Reply:
I'm going to revisit this later